misc/config_tools/static_allocators/intx.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2021 Intel Corporation. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'library'))
+import common, board_cfg_lib, lib.error, lib.lib
+
+LEGACY_IRQ_MAX = 16
+
+def get_native_valid_irq():
+    """
+     This is get available irq from board info file
+     :return: native available irq list
```

Comment:
The comments are not aligned with the `"""`. Is this intended?

Reply:
Removed

Reply:
OK